documentation/src/test/java17/org/hibernate/search/documentation/mapper/orm/projection/Author.java

```diff
@@ -29,10 +31,18 @@ public class Author {
 	@FullTextField(analyzer = "name", projectable = Projectable.YES)
 	private String lastName;
 
+	@VectorField(dimension = 4)
+	private byte[] embeddings;
+
+	@VectorField(dimension = 8)
+	private Float[] embeddingsFloat;
```

Comment:
Not sure it makes sense to accept `Float[]`... I hope you didn't have to go out of your way to support it?
`List<Float>` would be more useful IMO.

Reply:
+1 I'm waiting for HSEARCH-5006 to become available to open it 😆 (btw there are a few more todos in the changes where we probably can open other followup tickets like making the max vector size configurable etc)